src/components/SharedComponents/ScrollToTopButton.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { FixedButton } from './ScrollToTopButton.style';
+import { EventAction, EventCategory, trackEvent } from 'components/Analytics';
+
+const ScrollToTopButton = (props: {
+  showButton: boolean;
+  analyticsCategory: EventCategory;
+}) => {
+  const { showButton, analyticsCategory } = props;
+
+  const onClickHandler = () => {
+    trackEvent(analyticsCategory, EventAction.CLICK, 'Scroll to top button');
```

Comment:
maybe just 'Scroll to Top', so the label don't use that much space on the GA reports